Dokan/Dokan.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,9 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		27A983BF28662BEF00B59E94 /* CustomViewData.xib in Resources */ = {isa = PBXBuildFile; fileRef = 27A983BD28662BEF00B59E94 /* CustomViewData.xib */; };
+		27A983C028662BEF00B59E94 /* CustomViewData.swift in Sources */ = {isa = PBXBuildFile; fileRef = 27A983BE28662BEF00B59E94 /* CustomViewData.swift */; };
+		27A983C228662CC000B59E94 /* UIViewFromNIB.swift in Sources */ = {isa = PBXBuildFile; fileRef = 27A983C128662CBF00B59E94 /* UIViewFromNIB.swift */; };
```

Comment:
Pull request should be for a single reason. We don't need to add `CustomViewData` and `UIViewFromNIB`